deploy/docker/docker-compose.yml

```diff
@@ -42,7 +43,7 @@ services:
       mongodb:
         condition: service_healthy
     healthcheck:
-        test: curl 0.0.0.0:8080/identity/health_check
+        test: curl 0.0.0.0:${IDENTITY_SERVER_PORT}/identity/health_check
```

Comment:
Set a default value here as well so in case the .env file doesn't exist. This doesn't go haywire.
Would be especially helpful to provide just one file download for installation in dockerhub.
Same for other added variables.
Also the .env file feature only works when you use the docker-compose up command and does not work with docker stack deploy. Not that we have tested it with.

Reply:
I have added default values of the ports in the docker-compose file for the different services and have pushed the changes to my branch.